.github/workflows/test-sarif.yaml

```diff
@@ -0,0 +1,61 @@
+name: Test kube-linter SARIF output
+
+on:
+  - push
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # Checkout all repo history to make tags available for figuring out kube-linter version during build.
+          fetch-depth: 0
+
+      - name: Read Go version from go.mod
+        run: echo "GO_VERSION=$(grep -E "^go\s+[0-9.]+$" go.mod | cut -d " " -f 2)" >> $GITHUB_ENV
+      - name: Setup Go environment
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+
+      - name: Cache Go modules
+        uses: actions/cache@v2
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+
+      - name: Build binaries
+        run: make build
+
+      - name: Print kube-linter version
+        run: .gobin/kube-linter version
+
+      - name: Run kube-linter on a sample file with SARIF output
+        run: .gobin/kube-linter lint --format=sarif e2etests/testdata/inputs/splunk.yaml > results.sarif
```

Comment:
This file location should be reserve for Go tests written in the `e2etests` directory, IMO. I would suggest just creating another directory called `tests` or so in the root of the repo and putting this in a subdirectory under there.

Reply:
Thanks for suggestion. Moved to a new `tests/testdata` directory.